tools/devops/azure-pipelines.yml

```diff
@@ -10,6 +10,10 @@ jobs:
     name: 'Azure Pipelines'
     vmImage: 'internal-macos-10.15'
 
+  variables:
+  - name: ContinuousIntegrationBuild
+    value: true
+
```

Comment:
This is not the correct pipeline for CI, it is tools/devops/automation/build-pipeline.yml

I probably need to find what this pipeline is used, or even if it is used.

Reply:
I am removing this file here: https://github.com/xamarin/xamarin-macios/pull/14274 I understand is very misleading. You can add the variable at the top of  tools/devops/automation/build-pipeline.yml

Reply:
Thanks! I moved the variable declaration to the correct file.